dev-libs/spdlog/spdlog-1.4.2.ebuild

```diff
@@ -18,23 +18,20 @@ fi
 
 LICENSE="MIT"
 SLOT="0/1"
-IUSE="test"
+IUSE="test static-libs"
+RESTRICT="!test? ( test )"
 
 DEPEND="
 	>=dev-libs/libfmt-5.0.0
```

Comment:
this version of spdlog works with libfmt 5? If i'm not wrong this should be changed to dev-libs/libfmt:0/6, same for live ebuild.

thanks :)

Reply:
I opened an upstream issue asking exactly this, I couldn't find anything in the source that led me to believe it _wouldn't_ work with 5.x

https://github.com/gabime/spdlog/issues/1264

Reply:
oh, nice. Tomorrow I will try to take a look

Reply:
The author confirmed it as `>=dev-libs/libfmt-5.3.0`. I've set it to that